code/modules/martial_arts/combos/adv_first_aid/mend_bleeding.dm

```diff
@@ -0,0 +1,21 @@
+/datum/martial_combo/adv_first_aid/mend_bleeding
+	name = "Mend Artery"
+	steps = list(MARTIAL_COMBO_STEP_HELP, MARTIAL_COMBO_STEP_DISARM, MARTIAL_COMBO_STEP_GRAB, MARTIAL_COMBO_STEP_HARM)
+	explaination_text = "Mends an instance of internal bleeding in the targeted limb."
+
+/datum/martial_combo/adv_first_aid/mend_bleeding/perform_combo(mob/living/carbon/human/user, mob/living/target, datum/martial_art/MA)
+	if(!do_after(user, 10 SECONDS, TRUE, target))
+		to_chat(usr, "<span class='warning' you were interrupted. </span>")
+		return
+
+	if(!target.stat && ishuman(target))
```

Comment:
So as it turned out, removing the `ishuman` check kind of broke my code so I'll leave it there, doesn't hurt anything no? But I will remove the stat since yeah, wouldn't make sense to not work on unconscious person.